docs/docs/presets/joining-blocks.md

```diff
@@ -1,12 +1,12 @@
-# Presets >> Joining Blocks ||10
+# Presets >> Joining Blocks || 10
 
 The template system allows for a very granular control of how individual parts will be merged, overwritten or reorderd.
 
 As a preset you may want to add this to your layout.
 
 {% raw %}
 
-```
+```jinja2
```

Comment:
huh - what is that? I'm curious 🤗

Reply:
refractor (the syntax highlighting lib) doesn't have njk, but as nunjucks is a fork of jinja, jinja2 was the closest available alternative.